### DISCUSSION TITLE
Is it possible to simplify the rowHeight property of Cell

### DISCUSSION THREAD

**Discussion by ObjectiveCharm** [xmartlabs/Eureka#2198] [Q&A]
Hi developers👋
I wonder why the `rowHeight` property of Cell is of type `(()—>CGFloat)?` but not simple `CGFloat?`. Dose it is designed for any purpose I haven't recognised?
Thanks so much.

**Comment by mats-claassen** [comment]
Hi @ObjectiveCharm, rowHeight is a closure to allow for extra computation in case it is necessary. In most cases either a fixed height or automaticDimension will do but in some cases you might want to run some code to compute the height each time
